### ISSUE TITLE
Detected a defect share when mining with Nvidia 4070 supers with GRAM

### ISSUE THREAD

**Issue by logicminds** [Lolliedieb/lolMiner-releases#2059]
I am getting 100% defect shares when mining GRAM on my 4070 supers.  I have three of them (brand new) and they all show the same issue.  My 3080s do not have the issue so seems to be only with 4070 super or maybe the entire 40xx line.

I checked with the pool operator and they could find any issue so must be lol issue.

All 3 cards running same MMPOS version in different computers get the same issue. 

```
GPU 0: Detected a defect share. Check your undervolting & oc settings!
 Clk: 2010Mhz (600), P-State: 5, Mclk: 810Mhz (0), Pwr: 107W, FanSpeed: 40%, Volt: 0.940v, Temp: 49C, Junc Temp: 58C, Mem Temp: 36C
 
 er/0]: Connected to: ton.hashrate.to:4003                                                                                            │
│[2024-03-31 00:53:52][info][miner/0]:       Name          Speed  Pool  Shares   Best    Eff.  Power  CCLK  MCLK  Core  Junc   Mem  Fan                              │
│[2024-03-31 00:53:52][info][miner/0]:                      Mh/s  Mh/s  A/S/Hw  Share  Mh/s/W      W   MHz   MHz  Temp  Temp  Temp  Pct                              │
│[2024-03-31 00:53:52][info][miner/0]: GPU 0 Graphics De 4135.12  0.00  0/0/10    0.0  38.241  107.3  2010   810    49    57    36   40                              │
│[2024-03-31 00:53:52][info][miner/0]: ---------------------------                                                                                                   │
│[2024-03-31 00:53:52][info][miner/0]: Total             4135.12  0.00  0/0/10    0.0  38.241  107.3                                                                 │
│[2024-03-31 00:53:52][info][miner/0]: -----------------------------------------------                                                                               │
│[2024-03-31 00:53:52][info][miner/0]: Average speed (15s): 4134.95 Mh/s

lol version 1.8.5 beta
on pool connection mode: Ton stratum (developed by icemining.ca) (--ton-mode 6)
DNS over HTTPS resolve succeeded
Connected to ton.hashrate.to(95.216.76.150):4003  (TLS disabled)
Authorized worker: dddddd
Ton job with difficulty 68.72G and giver  received.
Start Mining...
Ton job with difficulty 68.72G and giver  received.
GPU 0: Detected a defect share. Check your undervolting & oc settings!
Ton job with difficulty 68.72G and giver  received.
Ton job with difficulty 68.72G and giver received.
Average speed (15s): 2147.72 Mh/s
^C
Closing lolMiner...
Ton job with difficulty 68.72G and giver  received.
GPU 0: solver stopped.
```

**Comment by jgonzis** [comment]
Yes that is because you were using 1.85_beta... that release hat a problem with the ADA, it was that information in the changelog, the new 1.86 has solved. So please use 1.86 or the old 1.84

**Comment by willansmith** [comment]
> I am getting 100% defect shares when mining GRAM on my 4070 supers. I have three of them (brand new) and they all show the same issue. My 3080s do not have the issue so seems to be only with 4070 super or maybe the entire 40xx line.
> 
> I checked with the pool operator and they could find any issue so must be lol issue.
> 
> All 3 cards running same MMPOS version in different computers get the same issue.
> 
> ```
> GPU 0: Detected a defect share. Check your undervolting & oc settings!
>  Clk: 2010Mhz (600), P-State: 5, Mclk: 810Mhz (0), Pwr: 107W, FanSpeed: 40%, Volt: 0.940v, Temp: 49C, Junc Temp: 58C, Mem Temp: 36C
>  
>  er/0]: Connected to: ton.hashrate.to:4003                                                                                            │
> │[2024-03-31 00:53:52][info][miner/0]:       Name          Speed  Pool  Shares   Best    Eff.  Power  CCLK  MCLK  Core  Junc   Mem  Fan                              │
> │[2024-03-31 00:53:52][info][miner/0]:                      Mh/s  Mh/s  A/S/Hw  Share  Mh/s/W      W   MHz   MHz  Temp  Temp  Temp  Pct                              │
> │[2024-03-31 00:53:52][info][miner/0]: GPU 0 Graphics De 4135.12  0.00  0/0/10    0.0  38.241  107.3  2010   810    49    57    36   40                              │
> │[2024-03-31 00:53:52][info][miner/0]: ---------------------------                                                                                                   │
> │[2024-03-31 00:53:52][info][miner/0]: Total             4135.12  0.00  0/0/10    0.0  38.241  107.3                                                                 │
> │[2024-03-31 00:53:52][info][miner/0]: -----------------------------------------------                                                                               │
> │[2024-03-31 00:53:52][info][miner/0]: Average speed (15s): 4134.95 Mh/s
> 
> lol version 1.8.5 beta
> on pool connection mode: Ton stratum (developed by icemining.ca) (--ton-mode 6)
> DNS over HTTPS resolve succeeded
> Connected to ton.hashrate.to(95.216.76.150):4003  (TLS disabled)
> Authorized worker: dddddd
> Ton job with difficulty 68.72G and giver  received.
> Start Mining...
> Ton job with difficulty 68.72G and giver  received.
> GPU 0: Detected a defect share. Check your undervolting & oc settings!
> Ton job with difficulty 68.72G and giver  received.
> Ton job with difficulty 68.72G and giver received.
> Average speed (15s): 2147.72 Mh/s
> ^C
> Closing lolMiner...
> Ton job with difficulty 68.72G and giver  received.
> GPU 0: solver stopped.
> ```

I also encountered the same problem. How did it work? Was it solved by using the 1.88 version?

**Comment by jgonzis** [comment]
Yes it is solved in 1.88
